.github/pull_request_template.md

```diff
@@ -16,10 +17,10 @@ if not applicable state N/A
 
 # Implementation tracking
 
-* [ ] validator tests
-* [ ] WPT tests: [LINK]()
```

Comment:
Are WPT tests subsumed under "user agent MUST"? (Put differently: in the wiki page should the sections on IDL, AAM, accname tests  be nested inside the section on "user agent MUST" test?)

Reply:
good idea!